toolchain/lower/context.h

```diff
@@ -140,7 +142,9 @@ class Context {
   llvm::raw_ostream* vlog_stream_;
 
   // The `FileContext`s for each IR that is involved in this lowering action.
-  Map<SemIR::CheckIRId, std::unique_ptr<FileContext>> file_contexts_;
+  using FileContexts =
```

Comment:
Don't we usually use `Store` as a suffix for the various store variants, rather than just pluralizing the type name? i.e.:

```suggestion
  using FileContextStore =
```

Reply:
:+1: done